src/qt/pivx/sendconfirmdialog.cpp

```diff
@@ -135,9 +138,12 @@ void TxDetailDialog::setData(WalletModel *model, WalletModelTransaction &tx){
     ui->textFee->setText(BitcoinUnits::formatWithUnit(nDisplayUnit, txFee, false, BitcoinUnits::separatorAlways));
 }
 
-void TxDetailDialog::acceptTx(){
-    this->confirm = true;
-    this->sendStatus = model->sendCoins(*this->tx);
+void TxDetailDialog::acceptTx()
+{
+    if (isConfirmDialog) {
```

Comment:
Would be good to throw an exception if this is false (to prevent anyone from calling it) and do the if/else in the event function `if (isConfirmDialog) acceptTx(); else accept();`

Reply:
done